### DISCUSSION TITLE
consulta

### DISCUSSION THREAD

**Discussion by andromeda2075** [cpp-review-dune/introductory-review#21] [Q&A]
He tratado de seguir la primera parte del [video de la clase de ayer](https://cpp-review-dune.github.io/videos/zoom/2022.html#sesion-46-cpp-review-dune-09-04-2022):
```console
gitpod /workspace/coupling/tutorials/perpendicular-flap/fluid-openfoam $ source /opt/OpenFOAM/OpenFOAM-v2112/etc/bashrc
bash: /opt/OpenFOAM/OpenFOAM-v2112/etc/bashrc: No such file or directory
```

**Comment by carlosal1015** [comment]
Probablemente no se llegó a instalar [`openfoam-com`](https://aur.archlinux.org/packages/openfoam-com) desde esta [URL](https://github.com/carlosal1015/aur/releases) que ya está compilado ya que tarde unas 5 horas en una computadora promedio.

Uno forma de conseguirlo sería:
```console
$ curl -LO https://github.com/carlosal1015/aur/releases/download/2022-04-14/openfoam-com-v2112-1-x86_64.pkg.tar.zst
$ sudo pacman -U openfoam-com-v2112-1-x86_64.pkg.tar.zst
$ source /opt/OpenFOAM/OpenFOAM-v2112/etc/bashrc
```
que descarga el paquete compilado, para luego instalarlo y finalmente activar variables y ejecutables de openFOAM.

> A futuro, calculo un par de semanas, el espacio de trabajo ya tendrá por defecto instalado los adaptadores de preCICE que incluye openFOAM y algunos módulos de dune que se requieran.
